app/assets/stylesheets/custome.scss

```diff
@@ -0,0 +1,5 @@
+@import "bootstrap-sprockets";
```

Comment:
may be *custom*

Reply:
Don`t understand what to improve here. What exactly i should do here?

Reply:
change file name)

Reply:
LOL, sorry i have understand)))) Willl rename)